concise-mid-tag.cddl

```diff
@@ -117,6 +94,7 @@ element-name-map = non-empty<{
   ? comid.model => model-type
   ? comid.layer => layer-type
   ? comid.index => index-type
+  ? comid.device-id => $device-id-type-choice
```

Comment:
Adding device-id to element-name seems wrong since it is only ever used in connection with identity claims. I'm not sure why this PR should affect identity-claims-map.